### PR TITLE
jarek/4932: use gradle makeDist to create docker version

### PR DESCRIPTION
**Pull request by jaroslawmalekcodete**

### CODE REVIEW

**Comment by jaroslawmalekcodete** [comment]
kick jenkins

**Comment by jaroslawmalekcodete** [comment]
kick jenkins

File: Dockerfile [hunk 1]
```diff
@@ -24,13 +24,23 @@ MAINTAINER Beaker Feedback <beaker-feedback@twosigma.com>
...
 
 ENV HOME /home/beaker
 
-RUN chown -R beaker:beaker /home/beaker
+RUN chown -R beaker:beaker /home/beaker && \
 
-RUN su -m beaker -c "cd /home/beaker/src && gradle realclean && gradle build"
+    su -m beaker -c "cd     /home/beaker/src/tmp && gradle realclean && gradle build" && \
+
+    su -m beaker -c "rm     /home/beaker/src/tmp/core/beaker-notebook*.zip || true" && \
```
Comment (scottdraves) [inline Dockerfile]: use `rm -f` instead of `|| true`

File: Dockerfile [hunk 2]
```diff
@@ -24,13 +24,23 @@ MAINTAINER Beaker Feedback <beaker-feedback@twosigma.com>
...
+
+    su -m beaker -c "cd     /home/beaker/src/tmp/core/config/builds/dist && gradle makeDist"  && \
+
+    su -m beaker -c "unzip  /home/beaker/src/tmp/core/beaker-notebook*.zip -d /home/beaker/src/"  && \
+
+    su -m beaker -c "mv     /home/beaker/src/beaker-notebook* /home/beaker/src/beaker_notebook"  && \
+
```
Comment (scottdraves) [inline Dockerfile]: no blank lines please

File: Dockerfile [hunk 3]
```diff
@@ -24,13 +24,17 @@ MAINTAINER Beaker Feedback <beaker-feedback@twosigma.com>
 #      Build      #
 ###################
 
-ADD . /home/beaker/src
+ADD . /home/beaker/src/tmp
```
Comment (scottdraves) [inline Dockerfile]: actually please keep this /home/beaker/src

File: Dockerfile [hunk 4]
```diff
@@ -24,13 +24,17 @@ MAINTAINER Beaker Feedback <beaker-feedback@twosigma.com>
...
 ENV HOME /home/beaker
 
-RUN chown -R beaker:beaker /home/beaker
-
-RUN su -m beaker -c "cd /home/beaker/src && gradle realclean && gradle build"
+RUN chown -R beaker:beaker  /home/beaker && \
+    su -m beaker -c "cd     /home/beaker/src/tmp && gradle realclean && gradle build" && \
+    su -m beaker -c "rm -f  /home/beaker/src/tmp/core/beaker-notebook*.zip" && \
+    su -m beaker -c "cd     /home/beaker/src/tmp/core/config/builds/dist && gradle makeDist" && \
+    su -m beaker -c "unzip  /home/beaker/src/tmp/core/beaker-notebook*.zip -d /home/beaker/src/" && \
```
Comment (scottdraves) [inline Dockerfile]: and unzip to /home/beaker/bin

**Pull request merged**